$NetBSD: patch-bg,v 1.3 2012/12/07 10:06:13 adam Exp $

this ends up going into non-const-able interfaces inside qt4

--- src/dbus/qdbusintegrator.cpp.orig	2011-03-30 05:19:07.000000000 +0000
+++ src/dbus/qdbusintegrator.cpp	2011-08-08 21:44:34.000000000 +0000
@@ -73,7 +73,7 @@
 static bool isDebugging;
 #define qDBusDebug              if (!::isDebugging); else qDebug
 
-Q_GLOBAL_STATIC_WITH_ARGS(const QString, orgFreedesktopDBusString, (QLatin1String(DBUS_SERVICE_DBUS)))
+Q_GLOBAL_STATIC_WITH_ARGS(QString, orgFreedesktopDBusString, (QLatin1String(DBUS_SERVICE_DBUS)))
 
 static inline QString dbusServiceString()
 { return *orgFreedesktopDBusString(); }
